common/djangoapps/student/views.py

```diff
@@ -855,6 +863,10 @@ def dashboard(request):
     valid_verification_statuses = ['approved', 'must_reverify', 'pending', 'expired']
     display_sidebar_on_dashboard = len(order_history_list) or verification_status in valid_verification_statuses
 
+    # Filter out any course enrollment course cards that are associated with fulfilled entitlements
```

Comment:
Can you clarify why we want to filter these out?

Reply:
Updated comment, don't want duplicate course runs